Renderer.js

```diff
@@ -0,0 +1,54 @@
+
+class Renderer {
+
+    renderMainUser() {
+        $('.user-container').empty()
+        const source = $('#user-template').html();
+        const template = Handlebars.compile(source);
+        const newHTML = template(mainUser);
+        $('.user-container').append(newHTML);
```

Comment:
Continuing from the above comment, this line can turn into:
`$('.user-container').empty().append(newHTML);`